pkg/rancher-desktop/pages/marketplace/details.vue

```diff
@@ -0,0 +1,352 @@
+<template>
+  <div class="extension">
+    <nuxt-child />
+    <div class="extension-content">
+      <div>
+        <Banner v-if="error" color="error" class="banner">
+          {{ error }}
+        </Banner>
+        <Banner v-if="response" color="info" class="banner">
+          {{ response }}
+        </Banner>
+        <div v-html="htmlContent" />
```

Comment:
NOTE: I wouldn't consider this to be trusted content, so we might need to port over the `v-clean-html` directive from Dashboard.